report/main.qmd

```diff
@@ -1 +1,166 @@
+
+When it comes to the residuals part of each site-specific model, it is suggested to compare four different correlation structure: independent, the first-order autoregressive process (AR(1)), the first-order moving-average process (MA(1)), and the combined autoregressive-moving-average processes (ARMA(p=1, q=1)). A GLS with an independent correlation structure is equivalent to a linear regression which assumes independence among residuals. A GLS with a AR(1) allows the residual at time point t to depend on the residual at the time point t-1. A GLS with a MA(1) is another way to capture temporal correlation among residuals, and ARMA(p=1, q=1) is the combination of AR(1) and MA(1) [@fox]. For each site, the GLS model with the smallest AIC values can be chosen as the final model. As a result, there will be three final models, one for each site, and the estimated coefficients of different interventions can be used for intervention effect assessment and comparison.
+
+One limitation of this approach is the challenge in statistically testing the differences in intervention effects across different sites. However, we could still learn differences between sites in a sense that maybe statistically significant effects are detected for some interventions in some sites but not in other sites. 
+
+## Conclusions
+It is recommended to use daily sales in percentage terms as response variable for the analysis, to better represent the relative consumption of zero-calorie and sugared beverages. Besides the tables showing the missingness of the dataset, the time series plots can be displayed to visualize the daily changes of zero-calorie drinks consumption, and the side by side boxplots can illustrate the relationship between the daily sales and various interventions at each site. 
+
+We suggest to implement GLS for each site to capture the association between changes in beverage consumption and the adopted interventions, allowing us to account for the dependence structure of residuals of the model.  Specifically, `Intervention` and `DofW` should serve as the predictor variables. The optimal model for each site can be determined by comparing the AIC across several candidate models, with each model specifying one of four different correlation structures. Differences in the effectiveness of various interventions and sites are identifiable by comparing the coefficients of the final models. 
+
+\newpage
+## References {.unnumbered}
+::: {#refs}
+:::
+
+\newpage
+## Statistical Appendix {.unnumbered}
+\beginAppendix
+
+```{r}
+# data missingness  by Site
+count_missing<- sale_data %>%
+    group_by(Site, Intervention) %>%
+    summarise(across(.cols = where(~ any(is.na(.x))), ~sum(is.na(.x))))
+
+# Table
+kable(count_missing, caption = "Number of missing observations for columns with missing data present by site and intervention type.")
+ 
+```
```

Comment:
This table is quite informative, but I think it would be better to use the `Intervention_updated` values instead of the `Intervention` values themselves as those are the names for the variable which you are using in your Formal Analysis section and the statistical models which you plan on creating.